test/cli/autocorrect-helpers/autocorrect-helpers.out

```diff
@@ -6,6 +6,10 @@ autocorrect-helpers.rb:4: Method `interface!` does not exist on `T.class_of(S1)`
 `
      4 |  interface!
         ^
+  Got T.class_of(S1) originating from:
+    autocorrect-helpers.rb:3:
+     3 |module S1
```

Comment:
Looks like the loc that we use for `<self>` when it's implicit is the `declLoc` of the containing definition.

Not great, but also not the thing I'm trying to change in this PR.

Reply:
Update: I decided to change this in this PR.